openshift-serverless-clients.spec

```diff
@@ -4,7 +4,7 @@
 %global product_name OpenShift Serverless
 %global golang_version 1.13
```

Comment:
If specific version of go is required here, then you can use `golang_version 1.13.15`.